functions/index.js

```diff
@@ -0,0 +1,21 @@
+const functions = require('firebase-functions');
+const fetch = require('node-fetch');
```

Comment:
Is there a reason you haven't used ES6 imports?

Reply:
Not easily supported by firebase environment